aiozk/session.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, servers, timeout, retry_policy, allow_read_only):
                 conn = await self.make_connection(host, port)
                 if not conn or (conn.start_read_only and not allow_read_only):
                     continue
+                else:
```

Comment:
[wglass/zoonado](https://github.com/wglass/zoonado/tree/master/zoonado) has the same mistake.